CODEOWNERS

```diff
@@ -166,6 +165,7 @@ Kconfig*                                  @tejlmand
 /samples/sensor/bh1749/                   @nrfconnect/ncs-cia
 /samples/sensor/bme68x_iaq/               @nrfconnect/ncs-cia
 /samples/bluetooth/                       @alwa-nordic @jori-nordic @carlescufi @KAGA164
+/samples/bluetooth/broadcast_config_tool/ @nrfconnect/ncs-audio
```

Comment:
Nice, but this PR does not remove the now unused audio folder.

Reply:
The folder no longer exist in my local repo. I think when you move all the files to another place that it will delete the folder as well.